### PR TITLE
Added additional FAQs

### PR DESCRIPTION
**Pull request by jtravee**
Signed-off-by: Jennifer Travinski <jennifer.travinski@suse.com>

Closes [#640](https://github.com/rancher-sandbox/rancher-desktop/issues/640)

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -42,6 +42,27 @@ https://docs.docker.com/desktop/
 **A:**
 -->
 
+<!-- #640 -->
+**Q: How can I restart Rancher Desktop?**
```
Comment (gunamata) [inline docs/faq.md:46]: May be `How to gracefully shutdown Rancher Desktop and all the services underneath?`

also, `File -> Exit`, `Tray icon exit` are basic features available for desktop apps. So, do we want to keep this question in faqs?

Comment (jtravee) [reply docs/faq.md]: I agree that it's basic functionality, so I'll remove it!

File: docs/faq.md [hunk 2]
```diff
@@ -42,6 +42,27 @@ https://docs.docker.com/desktop/
...
+
+**A:** Click **File** and then **Exit**, or right click on the tray icon at the bottom of the screen and choose to quit the app there.
+
+**Q: How can I perform a clean uninstall of Rancher Desktop?**
+
+**A:** You have two options for this procedure:
```
Comment (gunamata) [inline docs/faq.md:48]: actually, performing both the steps would produce a clean(er) uninstall.

File: docs/faq.md [hunk 3]
```diff
@@ -42,6 +42,27 @@ https://docs.docker.com/desktop/
...
+
+1. Click on the **Troubleshooting** tab in the Rancher Desktop UI. Once there, click on **Factory Reset**: this will reset to defaults and remove all user data such as Kubernetes, any caches, any configuration, logs, etc. After completing this step, Rancher Desktop should restart and start fresh.
+
+1. Click on **Add or Remove Programs** in your system settings, then uninstall Rancher Desktop there.
```
Comment (gunamata) [inline docs/faq.md:52]: uninstall process is different for different OSs and the steps are covered under the Installation section. we just link to those sections from here?

Comment (jtravee) [reply docs/faq.md]: Removed per feedback and instead linked to installation page.

File: docs/faq.md (hunk 2 above)
Comment (jtravee) [reply docs/faq.md]: Added link to installation page instead as advised.

File: docs/faq.md [hunk 4]
```diff
@@ -42,6 +42,19 @@ https://docs.docker.com/desktop/
 **A:**
 -->
 
+<!-- #640 -->
+**Q: How can I perform a clean uninstall of Rancher Desktop?**
+
+**A:** The uninstall process varies based on the operating system. For more information, please refer [here](https://docs.rancherdesktop.io/installation).
```
Comment (gunamata) [inline docs/faq.md:48]: Doing factory reset would still help in a cleaner uninstall than uninstall alone.  Uninstall alone would preserve the user  preferences from the previous install.

So, I would rephrase the answer to:

First perform a `Factory reset` (you can link to factory reset  section here) and then uninstall the app. The uninstall process varies based on the operating system. For more information, please refer [here](https://docs.rancherdesktop.io/installation).

Comment (jtravee) [reply docs/faq.md]: Revised per feedback, ty!

**Review by gunamata — APPROVED** [review]
LGTM
